uint8 brake_light_pos
uint64 timestamp 
uint64 source_time
